server/server/common/queries.py

```diff
@@ -460,3 +460,11 @@
     LIMIT @number
     RETURN text.text
 '''
+
+GLOSSARY = '''
+FOR gloss IN glossary
+    RETURN {
+        glossword: gloss.glossword,
+        description: gloss.description
+    }
```

Comment:
this is one way to do it. You can also use `KEEP(gloss, ['glossword', 'description'])` both are fine. Just as a tidbit